Bluetooth: Mesh: Various sensor fixes (#11776)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`bluetooth_mesh_sensor_server` sample:

  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.

>   * Added getter for the :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` sensor.

Reviewer: ```suggestion
  * Added a getter for the :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` sensor.
```

>   * Fixed an issue where the :c:var:`bt_mesh_sensor_time_since_presence_detected` sensor could report an invalid value when the time delta exceeds the range of the characteristic.

Reviewer: ```suggestion
  * Fixed an issue where the :c:var:`bt_mesh_sensor_time_since_presence_detected` sensor could report an invalid value when the time delta would exceed the range of the characteristic.
```

    * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.

  * Fixed an issue where the :ref:'bt_mesh_dtt_srv_readme' model could not be found for models spanning multiple elements.

>   * Fixed an issue where the :ref:'bt_mesh_sensor_srv_readme' model could add a corrupted marshalled sensor data into the Sensor Status message due to the fetched sensor value being outside of range.

Reviewer: ```suggestion
  * Fixed an issue where the :ref:'bt_mesh_sensor_srv_readme' model would add a corrupted marshalled sensor data into the Sensor Status message because the fetched sensor value was outside of range.
```

>     If the fetched sensor value is outside of range, the marshalled sensor data for that sensor won't be added to the Sensor Status message.

Reviewer: ```suggestion
    If the fetched sensor value is outside of range, the marshalled sensor data for that sensor will not be added to the Sensor Status message.
```
